app/views/podcasts/show.rss.builder

```diff
@@ -116,7 +116,8 @@ xml.rss "xmlns:atom" => "http://www.w3.org/2005/Atom",
         xml.pubDate ep.published_at.utc.rfc2822
         xml.link ep.url || ep.enclosure_url(@feed)
         xml.description { xml.cdata!(ep.description || "") }
-        xml.enclosure(url: ep.enclosure_url(@feed), type: ep.content_type(@feed), length: ep.file_size) if ep.media?
+        # TODO: may not reflect the content_type/file_size of replaced media
```

Comment:
Trying to untangle things - realized these content_type/file_size/duration methods aren't going to be accurate, if you just replaced/changed some files.

But I guess that's an edge case?  So I punted for now.